Python/Python.sublime-syntax

```diff
@@ -3,8 +3,10 @@
 name: Python
 file_extensions:
   - py
+  - py3
   - pyw
   - pyi
+  - pt
```

Comment:
There was a proposal for it [on the Python mailing list](https://mail.python.org/pipermail/python-list/2001-July/082404.html). I guess it (or something very similar) was implemented, because I have heard of the py3 file extension before.